int32 length
float64[] latitudes
float64[] longitudes
